deployment/maintain/rebase-helper.md

```diff
@@ -20,29 +20,32 @@ These programs are installed automatically as dependencies of rebase-helper.
 
 Note: rebase-helper is also available as EPEL-7 package. Feel free to use it on CentOS and RHEL systems.
 
-## How does the rebase-helper work?
+## How does rebase-helper work?
 Rebase-helper workflow can be summarized in following steps:
 
 - Downloads an archive with new sources.
 - Rebases downstream patches on top of the latest upstream version using git rebase command.
-- If patching was successful, it builds two sets of RPMs. One using the ``old`` sources and a second one using the ``new`` sources. Available build tools are ``mock``, ``rpmbuild``, ``fedpkg``.
-  - If ``new`` RPMs build fails, downloads logs like build.log and root.log
+- If patching was successful, it builds two sets of RPMs. One using the **``old``** sources and a second one using the **``new``** sources. Available build tools are ``mock``, ``rpmbuild``, ``fedpkg``.
+  - If **``new``** RPMs build fails, downloads logs like `build.log` and `root.log`
   - If the build is successful, downloads all available RPM packages.
-- If all builds succeed, rebase-helper runs several checkers, like ``pkgdiff``, ``rpmdiff``, ``abipkgdiff``. Rebase-helper compares old and new packages and reports results.
+- If all builds succeed, rebase-helper runs several checkers, like ``pkgdiff``, ``rpmdiff``, ``abipkgdiff``. Rebase-helper compares **old** and **new** packages and reports results.
 
 ## How to rebase your packages?
-Let's say, we want to rebase a package foobar from ``foobar-1.2.0`` to ``foobar-1.2.1``.
+Let's say we want to rebase a package foobar from ``foobar-1.2.0`` to ``foobar-1.2.1`` using rebase-helper:
 
-To update to the latest upstream version execute command:
+```sh
+# Change to the location of foobar.spec and other package components (cloned dist-git dir), e.g.
+cd $HOME/rpmbuild/REPOS/foobar
 
-```
+# Update to the selected upstream version
 rebase-helper 1.2.1
```

Comment:
What is this actually for?

Reply:
Actually, would you mind prefixing these commands with `$`?

Reply:
I played with that at one point, and determined that the syntax highlighting _really_ didn't handle that in a way that felt very readable. And it has the cost of making it a lot harder to cut-and-paste the commands, if someone wants to use them directly out of the document. I'll make the change, but IMHO of these two formats:

```sh
# Change to the location of foobar.spec and other package components (cloned dist-git dir), e.g.
$ cd $HOME/rpmbuild/REPOS/foobar

# Update to the selected upstream version
$ rebase-helper 1.2.1
```

```sh
# Change to the location of foobar.spec and other package components (cloned dist-git dir), e.g.
cd $HOME/rpmbuild/REPOS/foobar

# Update to the selected upstream version
rebase-helper 1.2.1
```
...The second is actually quite a bit clearer and more useful. (Again, purely IMHO.)

There's also `console` syntax highlighting, which is meant for console typescripts... but it doesn't style comments at all, and still fails to style the shell prompt usefully. Really it's more geared towards showing _output_ along with input, which we aren't doing.

```console
# Change to the location of foobar.spec and other package components (cloned dist-git dir), e.g.
$ cd $HOME/rpmbuild/REPOS/foobar

# Update to the selected upstream version
$ rebase-helper 1.2.1
```
...So, no real wins there, but a few losses.

Reply:
If we're to enhance the readability we'll do that site-wide. Feel free to submit an ehnancement (preferrably CSS or similar). As of now, we prefix all `commands` with `$` (see our [Contributing Guidelines](http://developer-portal.github.io/contributing/write-general-writing-hints.html)), so please do include it.

Also, FYI: the jekyl-built website looks different compared to Githubs-Markdown display.